core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsTableLoader.java

```diff
@@ -41,12 +42,14 @@
 
   public abstract GtfsTableContainer<T> load(
       InputStream inputStream,
-      ValidationContext validationContext,
+      GtfsFeedName feedName,
+      CurrentDateTime currentDateTime,
```

Comment:
Hmm, perhaps ValidationContext is fine to pass here since otherwise we would grow the argument list in the future.

The ValidationContext should be just extended with a function that picks a proper field based on its type.

public <T> T get(Class<T> clazz) {
    if (GtfsFeedName.class.isAssignableFrom(clazz)) {
      return (T) feedName();
    }
    if (CurrentDateTime.class.isAssignableFrom(clazz)) {
      return (T) now();
    }
    throw new IllegalArgumentException("Cannot find " + clazz.getCanonicalName() + " in validation context");
  }